### PR TITLE
js: Download Bootstrap on-the-fly during install instead

### PR DESCRIPTION
**Pull request by rht**
This PR:
- removes the need to attribute Bootstrap in a THIRDPARTY file
- reduces future diffs due to dependencies upgrade because they are no longer vendored entirely as files

### CODE REVIEW

**Comment by rht** [comment]
This should also make upgrading Bootstrap easier. @EwoutH if you notice, you miss some of the files in the first commit of #1100, e.g. the fonts and the npm.js.

**Comment by rht** [comment]
> @EwoutH if you notice, you miss some of the files in the first commit of https://github.com/projectmesa/mesa/pull/1100, e.g. the fonts and the npm.js.

I wasn't aware that you mentioned them as candidates for removal. Anyway, they will be auto-removed anyway since they are not included in the 4.6.1 zip file.

**Comment by EwoutH** [comment]
Maybe make the version a variable and use f-strings to incorporate the version in the paths, so that you can upgrade the version by changing a single variable.

**Comment by rht** [comment]
Yeah, I had in mind of putting explicit versions in the src path in mesa/visualization/templates/modular_template.html. This way, there is no collision between versions.

**Comment by rht** [comment]
I have isolated Bootstrap-specific files in the 2nd commit. Now, there is no problem with upgrading Bootstrap. But also notice the complication with having to update MANIFEST.in manually when upgrading Bootstrap.

This PR is ready for review.

File: setup.py [hunk 1]
```diff
@@ -20,6 +24,27 @@
 with open("README.rst", "rb", encoding="utf-8") as f:
     readme = f.read()
 
+# Ensure Bootstrap
+# Important: when you update the Bootstrap version, make sure to also update the
+# harcoded included files and versions in:
```
Comment (tpike3) [inline setup.py:29]: hardcoded is missing a "d" --- ha I kept subconsciously inserting the "d" and couldn't figure out the issue

**Comment by tpike3** [comment]
I feel  like in this comment and #1219 we are changing the squashed commits approach. I am fine with this as I think it will provided a cleaner historical record, but just want to call it out explicitly. Effectively, I would describe our commit approach as each commit should encapsulate a distinct update. Make sense?

**Comment by rht** [comment]
Yes, multiple commits are fine as long as they do orthogonal, atomic changes.

File: setup.py (hunk 1 above)
Comment (rht) [reply setup.py:29]: It's not caught by Codespell, ugh.

**Review by tpike3 — APPROVED** [review]
LGTM

**Pull request merged**